src/content/changelog/4-83-0-2021-01-11.md

```diff
@@ -0,0 +1,19 @@
+---
+title: '4.83.0: 2021-01-11'
+date: 2021-01-11T04:00:00.000Z
+version: 4.83.0
+changelog:
+  - API
+---
+
+### Added
+
+- The LKE [Kubernetes Cluster Recycle](/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-recycle) (POST /lke/clusters/{clusterId}/recycle) endpoint can now be used to recycle all of the Nodes in a Kubernetes cluster. Replacement Nodes are installed with the latest available patch for the Cluster's Kubernetes Version.
```

Comment:
Hi - it's me,  Heather! Just a note, these links need to be fully qualified or they will 404...like the last 2 API changelogs. https://linode.com/docs/api/...

Reply:
Ayyy thanks for the catch Heather!